package.json

```diff
@@ -32,6 +32,11 @@
         "title": "Update Package Dependencies",
         "icon": "$(cloud-download)",
         "category": "Swift"
+      },
+      {
+        "command": "swift.makeDebugConfig",
+        "title": "Generate launch configuration",
```

Comment:
We should use Title Case here for consistency (Generate Launch Configuration). Also, it's a bit weird that it's called Launch Configuration externally, but `debugConfig` internally (also in all the methods) 😅

Reply:
ok will fix